### PR TITLE
Update handshake chain migration in docker compose

### PR DESCRIPTION
**Pull request by MSevey**
When setting up a new portal I found that this command needs to be commented out in order for a new handshake node to start. 

It should stay in as a comment for existing portals

### CODE REVIEW

**Review by kwypchlo — COMMENTED** [review]
AFAIK handshake devs said that it can stay there since it will only run the migration if it's not be ran before. I don't think there's reason to comment it out.

**Comment by MSevey** [comment]
> AFAIK handshake devs said that it can stay there since it will only run the migration if it's not be ran before. I don't think there's reason to comment it out.

As I said in my initial comment, the reason to comment it out is it doesn't work. I'm setting up portals right now and handshake won't start until this is commented out.

**Review by kwypchlo — DISMISSED** [review]

**Review by ro-tex — APPROVED** [review]

**Comment by MSevey** [comment]
wait on this. testing a potential fix from the handshake devs

**Comment by MSevey** [comment]
Handshake added a fix, once https://github.com/handshake-org/hsd/pull/628 is merged I will update the dockerfile to point to the new commit that contains the fix.

**Comment by kwypchlo** [comment]
@MSevey https://github.com/handshake-org/hsd/pull/628 already merged :)

File: docker-compose.yml [hunk 1]
```diff
@@ -103,6 +103,7 @@ services:
     build:
       context: ./docker/handshake
       dockerfile: Dockerfile
+    # Migration.
```
Comment (kwypchlo) [inline docker-compose.yml:106]: this seems redundant

```suggestion
```

**Review by kwypchlo — APPROVED** [review]

**Review by ro-tex — DISMISSED** [review]

**Review by ro-tex — APPROVED** [review]

**Review by kwypchlo — APPROVED** [review]

**Pull request merged**